wiki/Tournaments/112/Summer_2019/en.md

```diff
@@ -0,0 +1,656 @@
+   - The player/team with correct answer **picks** first.
+        
+   - The player/team with incorrect answer **bans** first.
+        
+   - If both players/teams answer correctly, the winner is decided upon the first to answer according to the referee’s screen.
+        
+   - **From Quarterfinals onwards the team that bans first will ban one map, then the other team will ban two maps, then finally the first team will make their final ban.**
+
+4. During Group stages, each player can only ban 1 beatmap.
+
+   - **From Quarterfinals onwards, the number of bans increases to 2. However, each player/team may not ban more than one map of any mod bracket (excluding NoMod).**
+
+5. Beatmap selection will alternate between each player/team selecting a beatmap out of the mappool.
+
+6. Players map pick freely from any mod pool.
+
+   - In the case of a tie, the tiebreaker beatmap will be played.
+
+7. The results will be published via a statistics sheet.
+
+### Mappool Instructions
+
+1. There will be a new mappool for every stage.
+
+2. Each mappool consists of 5 brackets: NoMod, [Hidden](wiki/Game_Modifiers), [HardRock](wiki/Game_Modifiers), [DoubleTime](wiki/Game_Modifiers) and FreeMod.
+
+3. Each mappool has a specific size depending on the stage.
+
+   - The Group stage mappools will have 4 NoMod maps, 2 maps for each of the remaining mod brackets.
+        
+   - The Double Elimination stage mappools will have 5 NoMod maps, 3 maps for each of the remaining mod brackets.
+
+4. Each mappool has one tiebreaker.
+
+5. The NoFail mod may be selected on any map.
+
+6. The NoMod bracket will be played with no mods enabled at all (other than NoFail).
+
+7. The Hidden, HardRock and DoubleTime brackets will be played with the respective mod enabled by all players.
+
+8. The FreeMod bracket will have Free Mods enabled.
+
+   - Possible mod choices are Hidden, HardRock and Hidden+HardRock.
+
+9. The tiebreaker will be played under FreeMod conditions.
+
+   - When playing the tiebreaker, there are no mod requirements.
+
+### Scheduling Instructions
+
+1. Match schedule will be planned in accordance with weekly schedule-planning forms.
+
+2. Matches will be scheduled and announced according to the Turkey Time (UTC+3).
+
+3. Players may ask for a reschedule upon agreement with the opponent(s) if the current schedule does not work for them.
+
+4. Reschedules will only be considered if both players/teams agree to a time at least 24 hours before the first match in that particular stage.
+
+   - If players wish to change the scheduled time within 24 hours until the match, they have to seek for an available referee for the rescheduled time. **Participants cannot use this opportunity more than once.**
+
+5. If it is certain that participant(s) is/are unavailable for the scheduled time, they are obliged to inform the tournament staff beforehand. This is required in order not to waste the time of staff and other participants.  
+
+[flag_TR]: /wiki/shared/flag/TR.gif
```

Comment:
```suggestion
[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
```
Caption the flag, and make sure to Ascend Sort it. (A -Z) by the flag
e.g.
flag_ID
flag_JP
flag_KR
etc.